Eat_and_Greet/src/comps/Textbox/textbox.tsx

```diff
@@ -1,20 +1,39 @@
 import React, { useState } from 'react';
-import { TextInput, View } from 'react-native';
+import { TextInput, View , StyleSheet} from 'react-native';
 import styles from './style';
 
 interface TextBoxProps {
-  placeholder: string;
+  placeholder ?: string;
+  top ?: any;
+  left ?: any;
+  bottom ?: any;
+  right ?: any;
+  width ?: any;
+  height ?: any;
 }
 
-const TextBox = ( {placeholder} : TextBoxProps ) => {
+const TextBox = ( {placeholder, top, left, bottom, right, width, height} : TextBoxProps ) => {
 const [text, setText] = useState('');
-
 const handleTextChange = (userInput: string) => {
     setText(userInput);
 };
+const localstyle = StyleSheet.create({
```

Comment:
You already have a local style sheet, I think you should move this there? If you want to have them as separate objects, export both separately.